src/core/unit.c

```diff
@@ -6472,20 +6472,20 @@ int unit_arm_timer(
         return 0;
 }
 
-bool unit_is_filtered(Unit *u, char *const *states, char *const *patterns) {
+bool unit_filter(Unit *u, char * const *states, char * const *patterns) {
```

Comment:
Hm, `unit_filter` sounds more ambiguous for me.
Maybe, `passes_filter()` is better? Which is used in src/libsystemd/sd-device/device-monitor.c .